scripts/Dockerfile

```diff
@@ -14,13 +14,13 @@
 #       /home/klippy/.config/
 # For more Dockerfile examples with Klipper (and Octoprint) see:
 # https://github.com/sillyfrog/OctoPrint-Klipper-mjpg-Dockerfile
-FROM ubuntu:18.04
+FROM python:2
```

Comment:
Would 'python:3' now work here?

Reply:
It would, is python3 the preferred version now?